Lib/test/libregrtest/tsan.py

```diff
@@ -3,19 +3,24 @@
 
 TSAN_TESTS = [
     'test_code',
+    'test_compileall',
```

Comment:
Why this one? AFAIR it's long and shouldn't really stress any low-level threading issue.